recipes/pyalveo/meta.yaml

```diff
@@ -0,0 +1,74 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "pyalveo" %}
+{% set version = "1.0.3" %}
+{% set sha256 = "35cbdeb6423bfb6f9a2fb72d7707fdfe17a69fe0eda9a514c73b23604809c5fc" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - requests
+    - oauthlib
+    - requests-oauthlib
+  run:
+    - python
+
+test:
+  imports:
+    - pyalveo
+  requires:
+    - requests
+    - oauthlib
+    - requests-oauthlib
+    - python-dateutil
+
+about:
+  home: http://github.com/Alveo/pyalveo
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGLP.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: BSD-3-Clause
```

Comment:
I couldn't find any reference to a particular version of `BSD`. I've raised an issue upstream:

https://github.com/Alveo/pyalveo/issues/8

In the mean time, you can add the above link as a comment so we can resolve the license issue later in the feedstock once things are fixed upstream (unless it's fixed upstream before we merge this).